### PR TITLE
Fix api specs

### PR DESCRIPTION
**Pull request by lussoluca**
Fix couple of syntactic errors

### CODE REVIEW

File: api_proxy.yaml [hunk 1]
```diff
@@ -193,8 +193,7 @@ paths:
         in: path
         required: true
         description: The ID of the message.
-        schema:
-          $ref: "#/definitions/InstallationID"
+        type: string
```
Comment (gunzip) [inline api_proxy.yaml]: if you add min-length: 1 will produce a NonEmptyString

**Comment by digitalcitizenship** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Warnings</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:warning:</td>
      <td>

  Please include a Pivotal story at the beginning of the PR title (see below).
  </td>
    </tr>
  </tbody>
</table>



  Example of PR titles that include pivotal stories:

  * single story: `[#123456] my PR title`
  * multiple stories: `[#123456,#123457,#123458] my PR title`

    
<p align="right">
  Generated by :no_entry_sign: <a href="http://github.com/danger/danger-js/">dangerJS</a>
</p>

File: api_proxy.yaml (hunk 1 above)
Comment (lussoluca) [reply api_proxy.yaml]: this don't cause the generation of any type, is itn't?

Comment (gunzip) [reply api_proxy.yaml]: yes, actually types are only generated for schemas under the "definitions" swagger section (not for parameters)

**Pull request merged**